instrumentation/google.golang.org/grpc/otelgrpc/interceptor.go

```diff
@@ -73,10 +73,19 @@ func UnaryClientInterceptor(opts ...Option) grpc.UnaryClientInterceptor {
 		invoker grpc.UnaryInvoker,
 		callOpts ...grpc.CallOption,
 	) error {
+		cfg := newConfig(opts)
+		i := &InterceptorInfo{
+			Method: method,
+			Typ:    UnaryClient,
+		}
+		if cfg.Filter != nil && !cfg.Filter(i) {
```

Comment:
This applies to all similar logic, but doesn't this negate the activation of the filter?  `HealthCheck()` would return `true` so this would evaluate as false, stopping the logic from triggering.  Perhaps it worked differently before code suggestions?

I just happened to be looking for similar logic and pulled this code in to play with.

Reply:
As commented in https://github.com/open-telemetry/opentelemetry-go-contrib/pull/2672#issuecomment-1217311105, the issue is that the comment for `HealthCheck` is not right, and other filters are working as intended.

Reply:
@ymotongpoo can you put up a quick PR to fix the comment?  Thanks.